local_db.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
a helper script to set up the database locally as a docker container. This is the same as for the ARD, but the container has a different name